.github/workflows/build-ci.yml

```
@@ -29,6 +29,9 @@ jobs:
           distribution: 'temurin'
           java-version: '17'
 
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
```

Looks like these should all be version 4.

```suggestion
        uses: gradle/actions/setup-gradle@v4
```

---

v4 is an RC candidate right now, noted in https://github.com/deephaven/deephaven-core/pull/5898

---

Hmm okay - their readme suggests v4, so I assumed it was ready for prime time.